games-emulation/retroarch/retroarch-1.7.1.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="The reference frontend of libretro"
+DESCRIPTION="RetroArch is a frontend for emulators, game engines and media players. \
+It enables you to run classic games on a wide range of computers and consoles \
+through its slick graphical interface. It has advanced features like shaders, \
+netplay, rewinding, next-frame response times, and more!"
+HOMEPAGE="https://libretro.com"
+COPYRIGHT="2010-2018 The RetroArch Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/libretro/RetroArch/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="e9189c7da6b2ca0142528eca14f26104ee322f052f8609203d90b60938d8d3f3"
+SOURCE_DIR="RetroArch-$portVersion"
+PATCHES="retroarch-$portVersion.patchset"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	retroarch$secondaryArchSuffix = $portVersion
+	app:retroarch$secondaryArchSuffix = $portVersion
+	cmd:retroarch$secondaryArchSuffix = $portVersion
+	cmd:retroarch_cg2glsl$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	ffmpeg$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl2$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:which
+	cmd:git
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	"
+GLOBAL_WRITABLE_FILES="
+	$relativeSettingsDir/retroarch.cfg keep-old
+"
+
+BUILD()
+{
+	./fetch-submodules.sh
```

Comment:
I recommend adding `SOURCE_URI_` for each submodules, as it would allow one to build this package when a proxy is used. This is only a suggestion, so it's not strictly required.